### PR TITLE
Task #3

### LINKED ISSUE #3: Zadanie 3 (Island, Coordinates)
Napisz klasę `Island`, która będzie posiadała zmienną `Coordinates position_` oraz odpowiedni getter.

Klasa `Coordinates` ma określać współrzędne na mapie. Również ją napisz. Powinna przyjmować w konstruktorze 2 parametry `positionX`, `positionY` oraz operator porównania.

### PR DESCRIPTION
**Pull request by arafalski**
Resolves #3

### CODE REVIEW

File: shm/src/coordinates.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include "inc/coordinates.hpp"
```
Comment (kontrybutor) [inline shm/src/coordinates.cpp]: When PR with CMake will be merged, there won't be need to add `inc` prefix to `#include` I think.

File: shm/inc/island.hpp [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "coordinates.hpp"
+
+class Island {
+public:
+    Coordinates getPosition() { return position_; }
```
Comment (kontrybutor) [inline shm/inc/island.hpp]:
```suggestion
    Coordinates getPosition() const { return position_; }

```
getters's purpose is to retrieve data and they should not modify anything in the process.

File: shm/inc/coordinates.hpp [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+class Coordinates {
+public:
+    Coordinates(int positionX, int positionY)
+        : positionX_(positionX), positionY_(positionY) {}
+
+    bool operator==(const Coordinates& coordToCheck);
```
Comment (kontrybutor) [inline shm/inc/coordinates.hpp]:
```suggestion
    bool operator==(const Coordinates& coordToCheck) const;

```

File: shm/src/coordinates.cpp [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#include "inc/coordinates.hpp"
+
+bool Coordinates::operator==(const Coordinates& coordToCheck) {
```
Comment (kontrybutor) [inline shm/src/coordinates.cpp]:
```suggestion
bool Coordinates::operator==(const Coordinates& coordToCheck) const {

```

File: shm/src/coordinates.cpp (hunk 1 above)
Comment (arafalski) [reply shm/src/coordinates.cpp]: Yup. We can wait with that until the CMake is integrated with master. Then I can change to whatever it will be needed

**Review by Vicx95 — DISMISSED** [review]

**Review by Vicx95 — APPROVED** [review]

**Review by kontrybutor — APPROVED** [review]

**Pull request merged**